samtranslator/model/api/http_api_generator.py

```diff
@@ -7,6 +7,15 @@
 from samtranslator.open_api.open_api import OpenApiEditor
 from samtranslator.translator import logical_id_generator
 from samtranslator.model.tags.resource_tagging import get_tag_list
+from samtranslator.model.intrinsics import is_instrinsic
```

Comment:
`is_instrinsic` - typo

Reply:
the method name is mistyped, so we might have to reformat everywhere its called.